user/languages/_includes/modern-c-cpp-toolchain.md

```diff
@@ -0,0 +1,130 @@
+### Modern C/C++ and Toolchain Versioning
+
+If you need a more modern toolchain or more granular control over your tools, you will have to handle that yourself. Fortunately, the community has already whitelisted the [repositories](https://github.com/travis-ci/apt-source-whitelist/blob/master/ubuntu.json) and [packages](https://github.com/travis-ci/apt-package-whitelist/blob/master/ubuntu-precise) necessary for most such upgrades.
+
+#### GCC on Linux
+
+Ubuntu 12.04 shipped with GCC 4.6.3 and Ubuntu 14.04 shipped with GCC 4.8.2. If you need a more recent version (if you are working with C++11, you *must* do this; support for C++11 was not finished until GCC 4.9), you will want to follow these directions:
```

Comment:
I get it how must could translate into 'no pre c++11 support' but what I and @sxlijin (probably) meant was - you 'should definitely be using' c++11 now. I strongly support this argument because a lot of students I know have fallen back to not using newer(and better) language constructs just because their support was lacking somewhere in their build cycle which is unacceptable.